src/common/providers/identity.ts

```diff
@@ -326,26 +334,47 @@ export interface Credential {
   signInMethod: string;
 }
 
+/** Possible types of emails as described by the GCIP backend. */
+export type EmailType = "EMAIL_SIGN_IN" | "PASSWORD_RESET";
+
+/**
+ * The type of SMS message
```

Comment:
Can we include the comment below to the type comment above so it shows up in the reference docs?

Reply:
With salient details about valid values in the text preferably, thanks!